s2a/src/test/java/io/grpc/s2a/S2AChannelCredentialsTest.java

```diff
@@ -124,15 +124,15 @@ public void build_withUseMtlsToS2AWithLocalUid_success() throws Exception {
   }
 
   private static ChannelCredentials getTlsChannelCredentials() throws Exception {
-    String privateKeyPath = "src/test/resources/client_key.pem";
-    String certChainPath = "src/test/resources/client_cert.pem";
-    String trustBundlePath = "src/test/resources/root_cert.pem";
-    File privateKeyFile = new File(privateKeyPath);
-    File certChainFile = new File(certChainPath);
-    File trustBundleFile = new File(trustBundlePath);
+    InputStream privateKey = S2AChannelCredentialsTest.class
```

Comment:
You could pull `S2AChannelCredentialsTest.class.getClassLoader()` into its own variable and then each stream could fit on 1 line.

Reply:
Done in https://github.com/grpc/grpc-java/pull/11702/commits/adafbc14b2c2060c6687b59b6a71a727d84d052b